easybuild/easyconfigs/s/Sailfish/Sailfish-0.7.4-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Sailfish'
+version = '0.7.4'
+
+homepage = 'http://www.cs.cmu.edu/~ckingsf/software/sailfish/'
+description = """Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of
+ isoform abundances directly from a set of reference sequences and RNA-seq reads. """
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/kingsfordgroup/sailfish/archive/']
+sources = ['v%(version)s.tar.gz']
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+builddependencies = [
+    ('CMake', '3.3.2', '', ('GNU', '4.9.3-2.25')),
```

Comment:
this needs to change to `intel/2015b`, since this CMake pulls in `ncurses/5.9-GNU-4.9.3-2.25` as a dep, but the Boost below pulls in Python 2.7.10, which pulls in `ncurses/5.9-intel-2015b`, and both modules can't be loaded at the same time...

@ocaisa: This is an interesting case for you subtoolchain work...